modules/dev-tools/assets/js/editor/dev-tools.js

```diff
@@ -1,14 +1,102 @@
+/**
+ * @typedef {Object} Version
+ * @property {number} major1
+ * @property {number} major2
+ * @property {number} minor
+ * @property {string} build
+ */
 export default class extends elementorModules.editor.utils.Module {
-	onElementorLoaded() {
-		this.notifyDeprecated();
-	}
-
-	notifyDeprecated() {
+	notifyBackendDeprecations() {
 		// eslint-disable-next-line camelcase
 		const notices = elementor.config.dev_tools.deprecation.soft_notices;
 
 		Object.entries( notices ).forEach( ( [ key, notice ] ) => {
 			elementorCommon.helpers.softDeprecated( key, ...notice );
 		} );
 	}
+
+	/**
+	 * @param {string} version
+	 *
+	 * @return {Version}
+	 */
+	parseVersion( version ) {
+		const versionParts = version.split( '.' ),
+			versionPartsLength = versionParts.length;
+
+		if ( versionPartsLength < 3 || versionPartsLength > 4 ) {
+			throw new RangeError( 'Invalid Semantic Version string provided' );
+		}
+
+		const [ major1, major2, minor ] = versionParts,
+			result = {
+				major1: parseInt( major1 ),
+				major2: parseInt( major2 ),
+				minor: parseInt( minor ),
+			};
+
+		if ( versionPartsLength > 3 ) {
+			result.build = versionParts[ 3 ];
+		}
```

Comment:
Maybe deconstruct the `build` in line 31, and if it's defined just add that to the result object upon creation?

```suggestion
			result = {
				major1: parseInt( major1 ),
				major2: parseInt( major2 ),
				minor: parseInt( minor ),
				...( build ? { build } : {} ),
			};
```

Oe maybe:
```suggestion
			result = {
				major1: parseInt( major1 ),
				major2: parseInt( major2 ),
				minor: parseInt( minor ),
				build,
			};
```

This way you can avoid using a variable and just return the result
```
return {
	major1: parseInt( major1 ),
	major2: parseInt( major2 ),
	minor: parseInt( minor ),
	build,
}
```

Reply:
Handled.